conftest.py

```diff
@@ -16,6 +16,7 @@ def pytest_runtest_teardown():
 
 @pytest.fixture
 def app():
+    myapp.config['WTF_CSRF_ENABLED'] = False
```

Comment:
Could you also modify the name of the DB so that we don't lose the manual data when running tests?